graphql.md

```diff
@@ -0,0 +1,54 @@
+# GraphQL Development
+
+Our teams have extensive knowledge in developing GraphQL solutions and recomending number of packages depending on your
+use cases and languages.
+
+## Building GraphQL API in from scratch
+
+When building GraphQL API from scratch we recomend using reference GraphQL-js reference implementation which was 
+proven to be the most performant and have continous support from community. Entire development is currently backed by Linux foundation.
+
+### GraphQL Server
+
+For GraphQL Server we recomend using GraphQL-Express for exposing GraphQL APIs over the network 
+
+https://graphql.org/graphql-js/running-an-express-graphql-server
```

Comment:
This can be simplified once https://github.com/ardatan/graphql-tools/pull/2101 is merged